tests/strategy/test_interface.py

```diff
@@ -118,11 +107,7 @@ def test_get_signal_old_dataframe(default_conf, mocker, caplog, ohlcv_history):
     caplog.set_level(logging.INFO)
     mocker.patch.object(_STRATEGY, 'assert_df')
 
-    assert (False, False, None) == _STRATEGY.get_signal(
-        'xyz',
-        default_conf['timeframe'],
-        mocked_history
-    )
+    assert (False, False) == _STRATEGY.get_signal('xyz', default_conf['timeframe'], mocked_history)
```

Comment:
I think you'll need to rebase your branch on top of develop (or merge develop to it) to fix tests.

These changes don't seem related to your modifications - but break tests as get_signal returns a 3-element tuple now.